modules/KoreBuild.Tasks/module.targets

```diff
@@ -1,19 +1,21 @@
 <Project>
+
   <PropertyGroup>
     <PrepareDependsOn Condition=" '$(DisableDefaultTargets)' != 'true' ">GetToolsets;$(PrepareDependsOn)</PrepareDependsOn>
     <RestoreDependsOn Condition=" '$(DisableDefaultTargets)' != 'true' ">InstallDotNet;CheckPackageReferences;$(RestoreDependsOn)</RestoreDependsOn>
 
     <DisablePackageReferenceRestrictions Condition=" '$(DisablePackageReferenceRestrictions)' == '' ">false</DisablePackageReferenceRestrictions>
     <KoreBuildConfigFile Condition="'$(KoreBuildConfigFile)' == ''">$(RepositoryRoot)korebuild.json</KoreBuildConfigFile>
     <DependencyVersionsFile Condition="'$(DependencyVersionsFile)' == ''">$(RepositoryRoot)build\dependencies.props</DependencyVersionsFile>
+
   </PropertyGroup>
 
   <Target Name="EnableSkipStrongNames">
-    <Exec Command='"$(SkipStrongNamesExe)" -e "$(SkipStrongNamesXml)"' />
+    <Exec Command='dotnet "$(KoreBuildConsoleDll)" skip-strong-name enable --xml-file "$(SkipStrongNamesXml)"' />
```

Comment:
This will most likely work find, but FYI you can use the [RunDotNet task](https://github.com/aspnet/BuildTools/blob/dev/modules/BuildTools.Tasks/RunDotNet.cs) to avoid potential issues in PATH.
```
<RunDotNet Command='"$(KoreBuildConsoleDll)" skip-strong-name enable --xml-file "$(SkipStrongNamesXml)"' />
```